.github/workflows/build_test.yml

```diff
@@ -0,0 +1,112 @@
+name: Standard Build and Test
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - develop
+      - master
+  push:
+    branches:
+      - develop
+      - master
+
+jobs:
+  Housekeeping:
+    runs-on: ubuntu-latest
+    container:
+      image: svalinn/dagmc-ci-ubuntu-18.04-housekeeping:latest
+    steps:
+      - name: Install latest git
```

Comment:
?? we have to install git ??

Or is it just the newer version we need - maybe @Shimwell 's ghcr.io images have it already?

Reply:
The dockerhub and ghcr docker images appears to have the same git version.

```bash
sudo docker run -it svalinn/dagmc-ci-ubuntu-18.04-housekeeping:latest
git --version
 >> version 2.17.1
sudo docker run -it ghcr.io/svalinn/dagmc-ci-ubuntu-18.04-housekeeping:latest
git --version
 >> version 2.17.1
```

Reply:
They do. 
But we need 2.18 minimum so GitHub action use it for the checkout.
Otherwise it falls back on the "REST" api and just download a folder that does not contain git versioning info 

We need those for submodule and housekeeping

Reply:
Perhaps the ```actions/checkout@v2``` can also do submodules checkout but they are just off by default. I think it can be switched on (in the usage section of docs -> https://github.com/actions/checkout)